API/Repositories/Interfaces/IRepository.cs

```diff
@@ -6,36 +6,49 @@ public interface IRepository<T>
     // CRUD operations
 
     /// <summary>
-    ///  Adds a new entity to the database. Returns the added entity.
+    /// Adds a new entity to the database
     /// </summary>
+    /// <returns>The added entity</returns>
     Task<T> CreateAsync(T entity);
 
     /// <summary>
-    ///  Gets an entity from the database by its primary key. Returns null if not found.
+    /// Bulk inserts a collection of entities into the database
     /// </summary>
+    /// <returns>The added entities</returns>
+    Task<IEnumerable<T>> CreateAsync(IEnumerable<T> entities);
+
+    /// <summary>
+    /// Gets an entity from the database by its primary key
+    /// </summary>
+    /// <returns>The entity, or null if not found.</returns>
     Task<T?> GetAsync(int id);
 
     /// <summary>
-    /// Updates an entity, returning the number of rows affected.
+    /// Updates an entity
     /// </summary>
+    /// <returns>Number of rows affected</returns>
     Task<int> UpdateAsync(T entity);
 
     /// <summary>
-    ///  Deletes an entity from the database by its ID. If successful, returns the primary key of the entity, otherwise null.
+    /// Deletes an entity from the database by its ID
```

Comment:
I think `ID` should be `id` instead. This is the first time I've seen it in capitals.